### PR TITLE
fix: ignore nonce for call env

### PR DESCRIPTION
**Pull request by abhijeetbhagat**
addresses #6574

### CODE REVIEW

File: crates/rpc/rpc/src/eth/revm_utils.rs [hunk 1]
```diff
@@ -232,6 +232,8 @@ where
 ///  - `disable_block_gas_limit` is set to `true`
 ///  - `disable_eip3607` is set to `true`
 ///  - `disable_base_fee` is set to `true`
+///  - `disable_base_fee` is set to `true`
```
Comment (Rjected) [inline crates/rpc/rpc/src/eth/revm_utils.rs:235]: duplicate comment

**Review by Rjected — CHANGES_REQUESTED** [review]
hey, thanks for the contribution, is there a bug or regression that this solves?

**Comment by abhijeetbhagat** [comment]
> hey, thanks for the contribution, is there a bug or regression that this solves?

forgot to add the issue number this pr addresses. it addresses #6574

**Review by Rjected — APPROVED** [review]
ah, nice! this looks good to me

**Pull request merged**